templates/common/infra/bicep/core/host/container-app.bicep

```diff
@@ -5,14 +5,34 @@ param tags object = {}
 param containerAppsEnvironmentName string
 param containerName string = 'main'
 param containerRegistryName string
+
+@description('Minimum number of replicas to run')
+@minValue(1)
+param containerMinReplicas int = 1
+@description('Maximum number of replicas to run')
+@minValue(1)
+param containerMaxReplicas int = 10
```

Comment:
Setting to `10` here since this is the default value if not specified in ARM/Bicep.